examples/jsm/capabilities/WebGPU.js

```diff
@@ -6,20 +6,30 @@ if ( typeof window !== 'undefined' && isAvailable ) {
 
 }
 
+/**
+ * A utility module with basic WebGPU capability testing.
+ *
+ * @hideconstructor
+ */
 class WebGPU {
 
+	/**
+	 * Returns `true` if WebGPU is available.
+	 *
+	 * @return {boolean} Whether WebGPU is available or not.
+	 */
 	static isAvailable() {
 
 		return Boolean( isAvailable );
 
 	}
 
-	static getStaticAdapter() {
```

Comment:
@Mugen87 was there a reason behind removing `getStaticAdapter()` here?

Reply:
It was not used in the examples once so I assumed it was a left-over of earlier days.

If it has a relevance, we can of course revert the removal and document the method.